test/scylla_gdb/test_misc.py

```diff
@@ -190,4 +190,10 @@ def test_read_stats(gdb, sstable):
 def test_get_config_value(gdb):
     scylla(gdb, f'get-config-value compaction_static_shares')
 
+@pytest.mark.without_scylla
+def test_run_without_scylla(scylla_gdb):
+    # just try to load the scylla-gdb module without attaching to
+    # scylla
+    _ = scylla_gdb
```

Comment:
Please expand this comment: imagine somebody making a change to `scylla-gdb.py`, breaking this use-case and then this test fails and they come here to find out why. Currently, there is nothing here, pointing them to the right direction, the test is pretty cryptic.
We need to add an explanation here, that if this test fails, it is probably because `scylla-gdb.py` now cannot be loaded without debug symbols, and add instructions on how to keep this property.

Reply:
sure. added.